fury/utils.py

```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
     norm[faces[:, 2]] += n
     normalize_v3(norm)
     return norm
+
+def vertices_from_actor(actor):
+
+    """ This function will return vertices from actor
+    """
+    vtkPoint = actor.GetMapper().GetInput()
+    points = vtkPoint.GetPoints()
+    data = points.GetData()
+    return numpy_support.vtk_to_numpy(data)
+
+def compute_bounds(actor):
+
+    """ This function will compute the bounds
+    """
```

Comment:
same comments as above:
- No empty line
- start docstring with a verb
- finish with a period
- document input parameters and return